src/pages/FooterPages/ContributorPage/index.jsx

```diff
@@ -0,0 +1,89 @@
+import "../design.css";
+import "./index.css";
+
+import { Box, Typography, useMediaQuery } from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+import ContributorCard from "./ContributorCard";
+
+function Contributor() {
+  const [contributors, setContributors] = useState([]);
+  const isNonMobileScreen = useMediaQuery("(max-width: 800px)");
+
+  const getData = async () => {
+    // TODO: Add pagination
+    const res = await fetch(
+      "https://api.github.com/repos/narayan954/dummygram/contributors?per_page=100"
+    ); // TODO: Fix this to get all contributors
+
+    const data = await res.json();
+    const contributorsData = data.filter(
+      (contributor) =>
+        !contributor.login.includes(
+          "dependabot[bot]",
+          "deepsource-autofix[bot]"
+        )
+    );
+    setContributors(contributorsData);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  return (
+    <div className="contributor-container footer-page-para-color">
+      <div className="contributor-sub-container">
+        <span className="grad1 grad"></span>
```

Comment:
also, can you remind me, what're these spans for? the jelly background effect?

Reply:
> also, can you remind me, what're these spans for? the jelly background effect?
Yes

Reply:
yes